### ISSUE TITLE
MatType from a trace always returns 67 when it hits world on windows version 2024.03.21

### ISSUE THREAD

**Issue by ChillBoxServers** [Facepunch/garrysmod-issues#5832]
### Details
concommand.Add("tracetest", function (ply,cmd,args)
     PrintTable(ply:GetEyeTrace())
end)

When doing the above command or any type of trace that hits the world on windows version 2024.03.21 the MatType will always be 67, works on a linux server and a non updated server

### Steps to reproduce
1. install a fresh windows server
2. try to get anything other than a MatType of 67 from a trace on windows version 2024.03.21
![windows](https://github.com/Facepunch/garrysmod-issues/assets/66282823/bd5db5aa-bb3f-4bec-861e-f8772d8c70be)
![the_trace](https://github.com/Facepunch/garrysmod-issues/assets/66282823/800e218b-9dc8-40ec-8fe7-b27d70bd8946)
![linux](https://github.com/Facepunch/garrysmod-issues/assets/66282823/02c30cf2-a4d1-46e0-9963-a65214e69689)

**Comment by ChillBoxServers** [comment]
![image](https://github.com/Facepunch/garrysmod-issues/assets/66282823/1aa4cc4b-0f04-454c-8086-6788e11604e1)
This is just weird, i just started fresh and still get the same thing, installed a fresh server with steam cmd, factory reset/reinstalled gmod, when the trace is ran from the client its fine

**Comment by ChillBoxServers** [comment]
okay i used steam cmd and did app_update 4020 -beta x86-64 validate and now it works, something to do with the 32bit version?

**Comment by robotboy655** [comment]
No, the picture I provided is from "the 32bit version", the `dev` beta.

**Comment by DefineRust** [comment]
Can confirm this also happened to me,
![image](https://github.com/Facepunch/garrysmod-issues/assets/166741214/8cbb1eb2-9d02-4523-9b6c-6e47a828f248)
Completely fresh install, ChillBox can confirmed I literally just downloaded the dedi.

**Comment by robotboy655** [comment]
For some reason the materials are not loaded correctly on a clean windows srcds, things like `lua_run print(Material("gm_construct/grass_13"):GetString( "$basetexture"))` don't even work, but the issue does not happen on my local srcds builds for some inexplicable reason.

**Comment by robotboy655** [comment]
I figured it out, some DLLs need to be shipped with SRCDS which are currently not.

You can copy over `bin/stdshader_dx9.dll` to your srcds instance as a temporary fix, or use the `dev` beta as it will have the issue fixed soon.
